source/catalog.html

```diff
@@ -0,0 +1,159 @@
+                Соседи будут рады!
+              </p>
+              <p class="photo-item__likes">Нравится: 666</p>
+            </div>
+          </li>
+          <li class="photos-list__item photo-item">
+            <a href="#"><img class="photo-item__picture" width="280" height="180" src="" alt="Фотография Бориса"></a>
+            <div class="photo-item__description">
+              <p class="photo-item__author">Борис <span class="photo-item__timestamp">(20 часов назад)</span></p>
+              <p class="photo-item__text">
+                Здесь могла быть ваша цитата о высоком и вечном.
+              </p>
+              <p class="photo-item__likes">Нравится: 215</p>
+            </div>
+          </li>
+          <li class="photos-list__item photo-item">
+            <a href="#"><img class="photo-item__picture" width="280" height="180" src="" alt="Фотография Сергея"></a>
+            <div class="photo-item__description">
+              <p class="photo-item__author">Сергей <span class="photo-item__timestamp">(25 часов назад)</span></p>
+              <p class="photo-item__text">
+                Самая кубанская в мире!
+              </p>
+              <p class="photo-item__likes">Нравится: 4<p>
+            </div>
+          </li>
+          <li class="photos-list__item photo-item">
+            <a href="#"><img class="photo-item__picture" width="280" height="180" src="" alt="Фотография Максима"></a>
+            <div class="photo-item__description">
+              <p class="photo-item__author">Максим <span class="photo-item__timestamp">(40 часов назад)</span></p>
+              <p class="photo-item__text">
+                Где снег-то?
+              </p>
+              <p class="photo-item__likes">Нравится: 150</p>
+            </div>
+          </li>
+          <li class="photos-list__item photo-item">
+            <a href="#"><img class="photo-item__picture" width="280" height="180" src="" alt="Фотография Надежды"></a>
+            <div class="photo-item__description">
+              <p class="photo-item__author">Надежда <span class="photo-item__timestamp">(45 часов назад)</span></p>
+              <p class="photo-item__text">
+                Всем GM и взаимные лайки!
+              </p>
+              <p class="photo-item__likes">Нравится: 2560</p>
+            </div>
+          </li>
+        </ul>
+      </section>
+
+      <p class="page-main__description">
+        Затмите их всех! Выкладывайте все, что накопилось в телефоне!
+      </p>
+
+      <section class="editor">
+        <h2 class="visually-hidden">Редактор фотографий.</h2>
+        <img class="editor__photo" width="280" height="374" src="" alt="Фотография для редактирования">
+        <div class="editor__instruments">
+          <button class="editor__tools editor__tools--crop editor__tools--active" type="button">Обрезать</button>
+          <button class="editor__tools editor__tools--fill editor__tools--deactive" type="button">Заливка</button>
```

Comment:
```suggestion
          <button class="editor__tools editor__tools--fill editor__tools--inactive" type="button">Заливка</button>
```

неактивный = inactive